geometric/tests/test_run_json.py

```diff
@@ -0,0 +1,56 @@
+"""
+A set of tests for using the QCEngine project
+"""
+
+import numpy as np
+import json, os, shutil
+from . import addons
+import geometric
+
+@addons.using_qcengine
+@addons.using_rdkit
+def test_run_json():
+    # create a test input json file
+    qc_schema_input = {
+        "schema_name": "UFF gradient",
+        "schema_version": "v0.1",
```

Comment:
`"schema_version": 1,`

Reply:
This was copied from the `test_qcengine.py`, should we use a newer version of `qcengine` and also update that test?

Reply:
Ah, if you could fix that throughout it would be great. Future versions of `qcengine` will explicitly check for this.